fakeemail/web_server.py

```diff
@@ -9,8 +9,8 @@
 
 try:
     import json
-except:
-        import simplejson as json
+except BaseException:
```

Comment:
This would probably be better as `ImportError` as we're trying to establish whether we can import `json` or not.

Reply:
Done.